server/tests/cluster.rs

```diff
@@ -76,3 +82,170 @@ async fn replicated_loglet() -> googletest::Result<()> {
 
     Ok(())
 }
+
+#[test_log::test(restate_core::test)]
+async fn cluster_chaos_test() -> googletest::Result<()> {
+    let num_nodes = 3;
+    let chaos_duration = Duration::from_secs(20);
+    let mut base_config = Configuration::default();
+    base_config
+        .metadata_server
+        .set_kind(MetadataServerKind::Raft(RaftOptions {
+            raft_election_tick: NonZeroUsize::new(5).expect("5 to be non zero"),
+            raft_heartbeat_tick: NonZeroUsize::new(2).expect("2 to be non zero"),
```

Comment:
These made me go to the Raft options struct to look up how these work, and learn that they're multiples of the interval - which is 100ms by default. :-)